packages/contracts/package.json

```diff
@@ -7,16 +7,16 @@
     "----- EXTERNAL -----": "---------------------------",
     "init": "yarn tablegen && yarn build && yarn worldgen && yarn worldtypes",
     "prepare": "yarn tablegen && yarn worldgen && yarn worldtypes",
+    "devnode": "yarn mud devnode",
     "build": "forge clean && forge build",
-    "devnode": "mud devnode",
     "test": "echo todo",
     "dev": "yarn deploy",
-    "deploy": "yarn prepare && mud deploy-v2 --clean",
-    "deploy:testnet": "yarn prepare && mud deploy-v2 --profile=lattice-testnet",
+    "deploy": "yarn prepare && yarn mud deploy-v2",
```

Comment:
did you find that you needed `yarn` ahead of `mud` for it to work? my understanding of scripts is they run contextually inside the package/with the dependency binaries available - is that not true?

Reply:
I was under the impression it didn't work without yarn but let me double check that!